docs/DeveloperGuide.adoc

```diff
@@ -295,17 +295,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |mortuary manager |keep track of all bodies and fridges in a single dashboard using the dashboard command |look out for any outstanding work and keep myself up to speed
 
-|`* * *` |user |add a new person |
+|`* * *` |mortuary manager |have a dynamically updated dashboard |reduce errors as compared to updating manually
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |mortuary manager |sort the bodies by certain characteristics using the `sort` command |view bodies of a speciic category and generate statistics easily
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |mortuary manager |filter the bodies by certain characteristics using `filer` command |view bodies of a certain category and generate statistics easily
```

Comment:
'filer' spelling error

Reply:
Most of the diagrams in the Developer guide is not updated.